### ISSUE TITLE
/bin/gpg doesn't exist for CentOS and Fedora

### ISSUE THREAD

**Issue by sethcleveland** [DataDog/puppet-datadog-agent#260]
I found it in /usr/bin/gpg.

```
Notice: /Stage[main]/Datadog_agent::Redhat/Remote_file[DATADOG_RPM_KEY.public]/ensure: created
Error: /Stage[main]/Datadog_agent::Redhat/Exec[install-gpg-key]: Could not evaluate: Could not find command '/bin/gpg'
Notice: /Stage[main]/Datadog_agent::Redhat/Exec[cleanup-gpg-key]: Dependency Exec[install-gpg-key] has failures: true```

**Comment by sethcleveland** [comment]
Similar problem with /bin/test on CentOS

```Error: /Stage[main]/Datadog_agent::Redhat/Exec[cleanup-gpg-key]: Could not evaluate: Could not find command '/bin/test'
Notice: /Stage[main]/Seubase/Package[setuptools]/ensure: created```

**Comment by sethcleveland** [comment]
https://github.com/DataDog/puppet-datadog-agent/pull/259

**Comment by truthbk** [comment]
Thanks @sethcleveland will fix this soon, my bad for breaking it 😞

**Comment by truthbk** [comment]
Merged #259, closing issue.
